multipass-transfer: add page (#23234)
tldr-pages/tldr: pages/common/multipass-transfer.md

> # multipass transfer

Author: Not sure what the standard is here but your example contribution page had "docker-container-rm: add alias page" so i inferred that sub commands would have alias added but I've changed it back to just add page now.
it's a sub command of multipass

> Transfer files between the host and the virtual machine.

> > Relative paths on instance start from home folder (just like scp).

Reviewer: ```suggestion
> Note: Relative paths on an instance start from the user's home directory.
```

> More information: <https://canonical.com/multipass/docs/latest/reference/command-line-interface/transfer/>.

> - Transfer a file host to instance:

Reviewer: ```suggestion
- Transfer a file from the host to an instance:
```

`multipass transfer {{path/to/local_file}} {{instance_name}}:{{path/to/remote_directory}}`

> - Transfer a file from an instance to host:

Author: Bad grammar but it made it shorter/easier to read. Your call

> - Transfer a file from an instance to host:

Reviewer: ```suggestion
- Transfer a file from an instance to the host:
```

`multipass transfer {{instance_name}}:{{path/to/remote_file}} {{path/to/local_directory}}`

> - Transfer a directory from your local machine to an instance:

Reviewer: ```suggestion
- Transfer a directory from the host to an instance:
```
